src/components/main/HeaderSearchBox.jsx

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable import/no-unresolved */
+import React, { useState, useRef, useEffect } from 'react'
+import SearchIcon from '@mui/icons-material/Search'
+import { Box, Input } from '@mui/material'
+
+function HeaderSearchBox() {
+  const [isPopularVisible, setPoPularVisible] = useState(false)
```

Comment:
```suggestion
  const [isPopularVisible, setIsPopularVisible] = useState(false)
```
로 붙이는게 통상적인 룰입니다.
그리고 setPoPularVisible 대소문자가 잘못된것 같습니다